### PR TITLE
Fixes #489

### LINKED ISSUE #489: invalid arguments to math functions crash pixie
Calling a function like pixie.math/sin with a string or nil gives a segmentation fault

### PR DESCRIPTION
**Pull request by schaefed**
Raise runtime_error int to_float

### CODE REVIEW

**Comment by thomasmulvaney** [comment]
This looks great! Would you be able to add a test though please?

Thanks for working on this :-)

**Comment by schaefed** [comment]
Added a test to test-ffi.pxi. Not sure if this meets your criteria of an exhaustive test though...

File: tests/pixie/tests/test-ffi.pxi [hunk 1]
```diff
@@ -48,6 +48,14 @@
   (t/assert= 0.5 (asinf (sinf 0.5)))
   (t/assert= 1.0 (+ (powf (sinf 0.5) 2.0) (powf (cosf 0.5) 2.0))))
 
+(t/deftest test-invalid-float-argument
+  (try
+    (m/sin "nil")
+    (catch ex (t/assert= (type ex) RuntimeException)))
```
Comment (heyLu) [inline tests/pixie/tests/test-ffi.pxi]: Please use `t/assert-throws?`, so that the tests fail if no exception is thrown.

**Comment by schaefed** [comment]
Still not good enough?

**Comment by thomasmulvaney** [comment]
Looks good thanks! Sorry for the long wait but we didn't know you'd fixed that up.

**Pull request merged**